packages/gestalt/src/IconButtonFloating.js

```diff
@@ -0,0 +1,90 @@
+// @flow strict
+import { forwardRef, type Node } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import icons from './icons/index.js';
+
+type Props = {|
+  /**
+   * Specifies the `id` of an associated element (or elements) whose contents or visibility are controlled by IconButtonFLoating so that screen reader users can identify the relationship between elements. See the [Accessibility](#ARIA-attributes) guidelines for details on proper usage.
+   */
+  accessibilityControls?: string,
+  /**
+   * Indicates that IconButtonFloating hides or exposes collapsible components and expose whether they are currently expanded or collapsed. See the [Accessibility](#ARIA-attributes) guidelines for details on proper usage.
+   */
+  accessibilityExpanded?: boolean,
+  /**
+   * Indicates that this component controls the appearance of interactive popup elements, such as Dropdown or Modal. See the [Accessibility](#ARIA-attributes) guidelines for details on proper usage.
+   */
+  accessibilityHaspopup?: boolean,
```

Comment:
why is this a prop, and we don't just set it to true?

It should be a default right? I would remove this prop.

Reply:
haspopup doesn't apply for modals, only menus